### PR TITLE
Extract data from previous sessions page

### PR DESCRIPTION
**Pull request by benilovj**
- Extract the data from the 'Previous sessions' page markup
- Introduce two new view components (which are implemented as Nunjucks macros). This makes the markup a bit easier to understand:
  - a generic timeline item component
  - a warning component
- Get rid of a timeline variant that's not linked from anywhere in v6

### CODE REVIEW

File: app/data/cases.js [hunk 1]
```diff
@@ -105,7 +105,83 @@ module.exports = {
...
+        },
+        {
+          'type': 'Text message from Service User',
+          'receivedAt': '2021-04-22T13:00',
+          'contents': 'Hi Mark, got the email. I’ll be there. D.'
+        },
+        {
+          'type': 'Email to Service User',
+          'sentAt': '2021-04-10T13:00',
+          'truncatedContents': `Hi Dylan,
```
Comment (kelliedesigner) [inline app/data/cases.js:124]: Can we have a single `contents` prop and specify elsewhere the char length for truncating in the timeline?

File: app/data/cases.js [hunk 2]
```diff
@@ -105,7 +105,83 @@ module.exports = {
...
+          'outcome': null,
+        },
+        {
+          'type': 'Text message from Service User',
+          'receivedAt': '2021-04-22T13:00',
+          'contents': 'Hi Mark, got the email. I’ll be there. D.'
+        },
+        {
+          'type': 'Email to Service User',
+          'sentAt': '2021-04-10T13:00',
```
Comment (kelliedesigner) [inline app/data/cases.js:123]: `receivedAt` and `sentAt` dates could be merged into a single prop as we don't surface this detail to the user.

File: app/data/cases.js (hunk 1 above)
Comment (benilovj) [reply app/data/cases.js]: Good idea, done in  1e8cdf1

File: app/data/cases.js (hunk 2 above)
Comment (benilovj) [reply app/data/cases.js]: Agreed, done in https://github.com/ministryofjustice/hmpps-steel-thread/pull/101/commits/4d9bdfea4e71677e017015f29d22ac0559e2fa9e

**Comment by benilovj** [comment]
@kelliedesigner I've addressed your comments, if you could take another look that'd be good.

**Review by kelliedesigner — APPROVED** [review]
Looks good to me

**Pull request merged**